standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClientPreCatalog.java

```diff
@@ -569,9 +553,16 @@ private void open() throws MetaException {
                   + "Continuing without it.", e);
             }
           }
+        } catch (TTransportException e) {
+          tte = e;
+          if (LOG.isDebugEnabled()) {
```

Comment:
I know it is a copy but I think we should not have ... in the end of the message

Reply:
Agreed.